CHANGELOG.md

```diff
@@ -56,7 +57,6 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Added user consent screen
 - Public: Added callbacks that are triggered on user media submission if the feature is enabled. Note - This is a premium enterprise feature.
 - Internal: App component, Redux system, utils, HoCs & routers are now typed in TypeScript.
-- Internal: Use ScreenLayout component in Confirm screen.
```

Comment:
This line should not be deleted. 😅 The one on line 84 should be deleted

Reply:
Oh shit I was mistaken. Updated :point_up: